pkg/cmd/cli/describe/describer.go

```diff
@@ -363,17 +369,66 @@ func (d *ImageDescriber) Describe(namespace, name string) (string, error) {
 		return "", err
 	}
 
-	return describeImage(image)
+	return describeImage(image, "")
 }
 
-func describeImage(image *imageapi.Image) (string, error) {
+func describeImage(image *imageapi.Image, imageName string) (string, error) {
 	return tabbedString(func(out *tabwriter.Writer) error {
 		formatMeta(out, image.ObjectMeta)
 		formatString(out, "Docker Image", image.DockerImageReference)
+		if len(imageName) > 0 {
+			formatString(out, "Image Name", imageName)
+		}
+		formatString(out, "Parent Image", image.DockerImageMetadata.Parent)
+		formatString(out, "Layer Size", units.HumanSize(float64(image.DockerImageMetadata.Size)))
+		formatString(out, "Image Created", fmt.Sprintf("%s ago", formatRelativeTime(image.DockerImageMetadata.Created.Time)))
+		formatString(out, "Author", image.DockerImageMetadata.Author)
+		formatString(out, "Arch", image.DockerImageMetadata.Architecture)
+		describeDockerImage(out, &image.DockerImageMetadata.Config, &image.DockerImageMetadata.ContainerConfig)
 		return nil
 	})
 }
 
+func describeDockerImage(out *tabwriter.Writer, image, container *imageapi.DockerConfig) {
```

Comment:
Not now, although i think eventually we might want to show it.  Will remove.

----- Original Message -----
> >  		return nil
> >  	})
> >  }
> >  
> > +func describeDockerImage(out *tabwriter.Writer, image, container
> > *imageapi.DockerConfig) {
> 
> Do you need `container`?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2356/files#r30629299